src/applications/static-pages/school-resources/constants/announcements.json

```diff
@@ -1,233 +1,8 @@
-    "displayEndDate": "2020-05-22"
-  },
-  {    
-    "name": "107 Recertification Information for School Certifying Officials Webinar Page",
-    "url": "https://www.benefits.va.gov/gibill/resources/education_resources/school_certifying_officials/presentations.asp",
-    "date": "2020-04-22",
-    "displayStartDate": "2020-04-22",
-    "displayEndDate": "2020-05-22"
-  },
-  {    
-    "name": "Temporary relief from your GI Bill benefit debt",
-    "url": "https://www.facebook.com/gibillEducation/posts/10156986880046615?__tn__=-R",
-    "date": "2020-04-15",
-    "displayStartDate": "2020-04-15",
-    "displayEndDate": "2020-05-15"
-  },
-  {    
-    "name": "COVID-19 FAQs for School Certifying Officials (SCOs)",
-    "url": "https://benefits.va.gov/gibill/covid19faqs.asp",
-    "date": "2020-04-02",
-    "displayStartDate": "2020-04-02",
-    "displayEndDate": "2020-05-02"
-  },
-  {    
-    "name": "Important GI Bill Benefits Update - Congress Passes New Legislation due to COVID-19",
-    "url": "https://www.benefits.va.gov/gibill/#20200320",
-    "date": "2020-03-23",
-    "displayStartDate": "2020-03-23",
-    "displayEndDate": "2020-04-23"
-  },
-  {    
-    "name": "Yellow Ribbon Program Open Season for the 2020-2021 Academic Year",
-    "url": "https://www.benefits.va.gov/gibill/#20200319",
-    "date": "2020-03-23",
-    "displayStartDate": "2020-03-23",
-    "displayEndDate": "2020-04-23"
-  },
-  {    
-    "name": "Reschduled Webinar to March 26 – Office Hours – For State Certifying Officials (SCOs) to Discuss COVID-19 Impact",
-    "url": "https://www.benefits.va.gov/gibill/#202003132",
-    "date": "2020-03-13",
-    "displayStartDate": "2020-03-13",
-    "displayEndDate": "2020-04-13"
-  },
-  {
-    "name": "Webinar Slides Are Available for the VBA Education February SCO/SAA Office Hours",
-    "url": "https://www.benefits.va.gov/gibill/#20200313",
-    "date": "2020-03-13",
-    "displayStartDate": "2020-03-13",
-    "displayEndDate": "2020-04-13"
-  },
-  {
-    "name": "COVID-19 Virus Concerns and How It Affects Your Benefits",
-    "url": "https://www.benefits.va.gov/gibill/#20200312",
-    "date": "2020-03-12",
-    "displayStartDate": "2020-03-12",
-    "displayEndDate": "2020-04-12"
-  },
-  {
-    "name": "Recertification for Schools Impacted by Colmery Act, Section 107",
-    "url": "https://www.benefits.va.gov/gibill/#20200310",
-    "date": "2020-03-10",
-    "displayStartDate": "2020-03-10",
-    "displayEndDate": "2020-04-10"
-  },
-  {
-    "name": "Suspension of VA Benefits to Five Schools for Deceptive Practices",
-    "url": "https://www.benefits.va.gov/gibill/#20200309",
-    "date": "2020-03-09",
-    "displayStartDate": "2020-03-09",
-    "displayEndDate": "2020-04-09"
-  }
-]
+    "name": "Yellow Ribbon Program Open Season June 15-June 30",
+    "url": "https://www.benefits.va.gov/gibill/#20210526",
+    "date": "2021-05-26",
+    "displayStartDate": "2021-05-26",
+    "displayEndDate": "2021-06-26"
+  ]
```

Comment:
You need a closing curly brace } for the JSON data. It might be easier if you left a couple of old ones in there for reference.